### PR TITLE
Test builder with travis

### PR DESCRIPTION
**Pull request by alexander-schranz**
Added a travis ci script which:

 - tests the `sulu:build` command
 - tests the `phpunit` setup
 - lint the `twig` files
 - lint the `yaml` files

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+cache:
+  directories:
+    - "$HOME/.composer/cache/files"
+
+matrix:
+  include:
+    - php: 5.5
+      env:
+        - COMPOSER_FLAGS="--prefer-dist --no-interaction"
```
Comment (alexander-schranz) [inline .travis.yml]: this is not set to lowest as we want to test the actual sulu version and not 1.6.0

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  - composer info -i
+  # Create database for build test
+  - bin/adminconsole doctrine:database:create
+  # Create database for unit test
+  - bin/adminconsole doctrine:database:create --env=test
+
+script:
+  # Test if installation works
+  - bin/adminconsole sulu:build dev --no-interaction
+  # Test if PHPUnit isntallation works
```
Comment (danrot) [inline .travis.yml]: `installation`

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  - SYMFONY_ENV=test bin/simple-phpunit
+  # Lint Twig files
+  - bin/adminconsole lint:twig app/Resources/views
+  # Lint Yaml files
+  - bin/adminconsole lint:yaml app/config
+
```
Comment (danrot) [inline .travis.yml]: Looks like a new line too much

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+language: php
+
+cache:
+  directories:
+    - "$HOME/.composer/cache/files"
+
+matrix:
+  include:
+    - php: 5.5
```
Comment (danrot) [inline .travis.yml]: 5.5 doesn't seem to work anymore :-)

**Review by danrot — CHANGES_REQUESTED** [review]

**Review by danrot — APPROVED** [review]

**Pull request merged**